### ISSUE TITLE
Mouse/keyboard check reports false positives

### ISSUE THREAD

**Issue by jgeudens** [pluyckx/kam#19]
When my laptop was idle for the evening, it didn't go to sleep like usual. When examining the logs, I noticed that the mice and key event were logged. The strange thing is that the keyboard and mice activity is detected at the same time.

```
2019-01-12 20:28:48 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True
2019-01-12 20:28:48 [MiceCheck:config] mice = ['event5', 'mouse0', 'mouse1'];  // True
2019-01-12 20:28:48 [KeyboardCheck:check] read from:  = [];  // True
2019-01-12 20:28:49 [MiceCheck:check] read from:  = [];  // True
2019-01-12 20:39:49 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True
2019-01-12 20:39:49 [MiceCheck:config] mice = ['event5', 'mouse0', 'mouse1'];  // True
2019-01-12 20:39:49 [KeyboardCheck:check] read from:  = [];  // True
2019-01-12 20:39:49 [MiceCheck:check] read from:  = [];  // True
2019-01-12 20:47:49 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True
2019-01-12 20:47:49 [MiceCheck:config] mice = ['event5', 'mouse0', 'mouse1'];  // True
2019-01-12 20:47:49 [KeyboardCheck:check] read from:  = [];  // True
2019-01-12 20:47:49 [MiceCheck:check] read from:  = [];  // True
2019-01-12 20:55:49 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True
2019-01-12 20:55:50 [MiceCheck:config] mice = ['event5', 'mouse0', 'mouse1'];  // True
2019-01-12 20:55:50 [KeyboardCheck:check] read from:  = [];  // True
2019-01-12 20:55:50 [MiceCheck:check] read from:  = [];  // True
2019-01-12 20:56:50 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True
2019-01-12 20:56:50 [MiceCheck:config] mice = ['event5', 'mouse0', 'mouse1'];  // True
2019-01-12 20:56:50 [KeyboardCheck:check] read from:  = [];  // True
2019-01-12 20:56:50 [MiceCheck:check] read from:  = [];  // True
2019-01-12 21:01:50 [KeyboardCheck:config] keyboards = ['event7', 'event8', 'event6', 'event4'];  // True

```

**Comment by pluyckx** [comment]
Can you upload a full log (without filtering on *True*)?

**Comment by pluyckx** [comment]
I think I know what is causing the issue. When you stop using your device, could you open a terminal and execute `udevadm monitor`. I think udev is generating output and this forces the *mouse* and *keyboard* check modules to reload.

Then check periodically if there is output. If there is, let me know and if useful send the output to me.

**Comment by pluyckx** [comment]
The issue is fixed in v1.1.2
